pyfar/classes/audio.py

```diff
@@ -1348,26 +1348,28 @@ def _assert_match_for_arithmetic(data: tuple, domain: str, division: bool,
     # properties that must match
     sampling_rate = None
     n_samples = None
-    fft_norm = 'none'
+    # None indicates that no audio object is yet involved in the operation
+    # it will change upon detection of the first audio object
+    fft_norm = None
     times = None
     frequencies = None
     audio_type = type(None)
     cshape = ()
 
     # check input types and meta data
-    found_audio_data = False
-    for n, d in enumerate(data):
+    n_audio_objects = 0
+    for d in enumerate(data):
```

Comment:
```suggestion
    for _, d in enumerate(data):
```
otherwise sth breaks